tests/integration/session/tls_handshake.rflx

```diff
@@ -89,24 +87,24 @@ package TLS_Handshake is
 
    -- Key Exchange Messages
 
-   type Protocol_Version is (
-      TLS_1_0 => 16#0301#,
-      TLS_1_1 => 16#0302#,
-      TLS_1_2 => 16#0303#,
-      TLS_1_3 => 16#0304#
-   ) with Size => 16, Always_Valid;
+   type Protocol_Version is
+      (TLS_1_0 => 16#0301#,
+       TLS_1_1 => 16#0302#,
+       TLS_1_2 => 16#0303#,
+       TLS_1_3 => 16#0304#)
+   with Size => 16, Always_Valid;
 
    type Legacy_Session_ID_Length is range 0 .. 32 with Size => 8;
 
    type Cipher_Suites_Length is range 2 .. 2**16 - 2 with Size => 16;
 
-   type Cipher_Suite is (
-      TLS_AES_128_GCM_SHA256 => 16#1301#,
-      TLS_AES_256_GCM_SHA384 => 16#1302#,
-      TLS_CHACHA20_POLY1305_SHA256 => 16#1303#,
-      TLS_AES_128_CCM_SHA256 => 16#1304#,
-      TLS_AES_128_CCM_8_SHA256 => 16#1305#
-   ) with Size => 16, Always_Valid;
+   type Cipher_Suite is
+      (TLS_AES_128_Gcm_SHA256       => 16#1301#,
+       TLS_AES_256_Gcm_SHA384       => 16#1302#,
+       TLS_Chacha20_Poly1305_SHA256 => 16#1303#,
+       TLS_AES_128_Ccm_SHA256       => 16#1304#,
+       TLS_AES_128_Ccm_8_SHA256     => 16#1305#)
```

Comment:
`GCM` and `CCM` are abbreviations. I'm not sure if we should use `ChaCha`.

Reply:
Changed in 354f91e. I think it makes sense to call it correctly `ChaCha20`.